Summer Ball 2020 [WIP test merge]

CitrusGender opened the pull request:
yeah this one is pretty huge and has a lot of stuff in it


I don't know where to begin but it compiles locally so it works. 

Thanks to all the mappers/spriters/coders for making this happen. This was a huge project and probably the best one to date.

DO NOT MERGE, CURRENTLY A WORK IN PROGRESS AND ONLY TO BE TESTMERGED

optimumtact committed:
whops

optimumtact committed:
Give it more time

optimumtact committed:
More changes

optimumtact committed:
You can no longer cut down a tree

optimumtact committed:
Disable event picks

optimumtact committed:
Disable breeding in event mode
fix phpism

optimumtact committed:
Disable smugglers satchels in event mode

EOBGames committed:
Sets up silicons to be safely played at the ball (#25)

* partimov

* whoops I forgot a "

ArcaneMusic committed:
Alright, lets see how much of a pain this is

ArcaneMusic committed:
Grammar

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

# Conflicts:
#	code/modules/mob/living/silicon/robot/robot_modules.dm

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

# Conflicts:
#	code/modules/mob/living/silicon/robot/robot_modules.dm

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

ishitbyabullet committed:
does it work yet

i fucking hate github.

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

Commit:
Merge branch 'eventmode' into summerball2020

# Conflicts:
#	code/modules/atmospherics/gasmixtures/gas_mixture.dm
#	code/modules/vending/_vending.dm

adds orange's eventmode and json for summer ball.

Commit:
forgot extra endif

Commit:
adds map, changes one extra thing in gas_mixture.dm

Commit:
see map bus for changelog

Commit:
Merge branch 'SummerBallShit' of git://github.com/ArcaneMusic/tgstation-1

EOBGames committed:
puzzle shit

ishitbyabullet committed:
Merge pull request #33 from EOBGames/jungletemplecontent

Some Puzzling Changes

Fikou committed:
adds cat roomba and adminbus content (#30)

* weed

* catroomba

* fuck bees

* a

* 114

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

Commit:
updates map, see mapper bus for changelog

MMMiracles committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summer_ball_2020

# Conflicts:
#	code/modules/mob/living/silicon/robot/robot_modules.dm

MMMiracles committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summer_ball_2020

# Conflicts:
#	_maps/map_files/BallMap/summer_ball.dmm

actions-user committed:
Automatic changelog compile [ci skip]

EOBGames committed:
the great casino rework, part 1 (#34)

actions-user committed:
Automatic changelog compile [ci skip]

ArcaneMusic committed:
Pin the tail on the corgi content (#26)

* Alright, lets see how much of a pain this is

* Grammar

* Tailsprite now properly bullseyes

actions-user committed:
Automatic changelog compile [ci skip]

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge branch 'master' of https://github.com/CitrusGender/tgstation

# Conflicts:
#	html/changelogs/.all_changelog.yml

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

EOBGames committed:
More Map Changes, Wardrobe Stuff (#36)

* aaaaaa

* jesus fuck that's a lotta clothes

* oops

* unfucks vendors

EOBGames committed:
casino shell (#38)

trollbreeder committed:
Keep my filthy blood piles if i want them, please! (#37)

ishitbyabullet committed:
HEALTH - TEARS

There were UFA all over the airport and civilians were being moved out. Looking at it one way, shutting down the airport for their escape was a weird sort of compliment, but one I didn't need.

ishitbyabullet committed:
Merge pull request #39 from ishitbyabullet/baldchanges

HEALTH - TEARS

ishitbyabullet committed:
Howdy, Hotlander!

I am washed - I am washed - I am washed - I am washed - I am washed in the blood of the lamb!

CitrusGender committed:
Update air.dm

>web edits
make it so that atmos can't fire during the ball.

EOBGames committed:
future club is a based song (#41)

listen to it

ishitbyabullet committed:
Howdy, Hotlander! (#40)

I am washed - I am washed - I am washed - I am washed - I am washed in the blood of the lamb!

ishitbyabullet committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into baldchanges

ishitbyabullet committed:
verbalase beatbox

Ya’ll gone make me + Beatbox
Ya’ll gone make me + Beatbox
Ya’ll gone make me + Beatbox
Ya’ll gone make me + Beatbox

ishitbyabullet committed:
The town layout (#42)

* Howdy, Hotlander!

I am washed - I am washed - I am washed - I am washed - I am washed in the blood of the lamb!

* verbalase beatbox

Ya’ll gone make me + Beatbox
Ya’ll gone make me + Beatbox
Ya’ll gone make me + Beatbox
Ya’ll gone make me + Beatbox

AnturK committed:
Mafia code (#31)

* Mafia initial version.

* Removes debug stuff

* tgui build

* Fixes, adds visible timer, adds setups, makes debugging easier

* Makes votes more visible

* tgui build

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	html/changelogs/.all_changelog.yml

ishitbyabullet committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into baldchanges

ishitbyabullet committed:
ONE IN THE PIPE!!

Stay right there-  I'll make this quick!

ishitbyabullet committed:
Merge pull request #44 from ishitbyabullet/baldchanges

guys wouldnt it be funny if i omega griefed the ball

GreyTide2108 committed:
Prophunt Area Code / Arena Maps (#45)

Just code and .dmms

GreyTide2108 committed:
Prophunt Arena Map Addition (But Right This Time™) (#47)

* Prophunt Area Code / Arena Maps

* Prophunt Arena

Adds Control Room, Arena, Ready Area, Lobby, and Maintinence Spectator Room / Bar.

* Update summer_ball.dmm

Commit:
multi-z works now thanks to anturk

Commit:
adds blackjack tables to ball map, adds new icons for signs (big/small), adds new turf for the boardwalk, adds blackjack icons.

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	html/changelogs/.all_changelog.yml

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

# Conflicts:
#	tgui/packages/tgui/public/tgui.bundle.js

AnturK committed:
Arena stuff (#48)

* Simple option to delete objects leaving arena.

* Adds few more configurable things.

* Let me pr github

actions-user committed:
Automatic changelog compile [ci skip]

Commit:
whitelists security, adds nonsmoke teleportation scroll, changes boxes to have said scroll, removes all roles but security/borg/assistant.

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

Commit:
done

Commit:
Merge branch 'ishitbyabullet-baldchanges' into summerball2020
t push origin summerball2020

Commit:
adds jerry's map again

GreyTide2108 committed:
Edit to Prop Hunt Readme

ishitbyabullet committed:
Merge pull request #50 from GreyTide2108/summerball2020

Edit to Prop Hunt Readme

ishitbyabullet committed:
breaking benjamin only the strongest will survive

master chief!!!

ishitbyabullet committed:
Merge pull request #51 from CitrusGender/I_hate_github

GUYS, I AM GOING TO GRIEF THE BALL

EOBGames committed:
spirit of freedom (#52)

Commit:
updates map with jerry's changes

Pumpkinoe committed:
Medbay completion

Completes the """Medbay"""" if i did this right

AnturK committed:
no replacement for turfs

AnturK committed:
skips light source churn

ishitbyabullet committed:
Cruo StragaraNa Malactose

Crudox crou!

ishitbyabullet committed:
Merge pull request #54 from CitrusGender/I_hate_github

fikou has mapped bababooey

AnturK committed:
glass tables don't break

ishitbyabullet committed:
You gotta be fucking kidding, necropolis

LEGION SHALL NEVER WAKE UP NOW

ishitbyabullet committed:
round two

here we go

AnturK committed:
whoops

ishitbyabullet committed:
Merge pull request #53 from AnturK/misc_event_stuff

Misc event stuff

EOBGames committed:
RAMPAGE!!

sugar water and zima != cancer medication

ishitbyabullet committed:
Merge pull request #56 from EOBGames/summerball6

RAGE CAGE

ishitbyabullet committed:
Extra stuff

Adding this incase.

ishitbyabullet committed:
Merge pull request #55 from CitrusGender/I_hate_github

Necropolis gate

ishitbyabullet committed:
Mt. Labyrinthian

You'll never leave this place alive.

ishitbyabullet committed:
Merge pull request #57 from CitrusGender/I_hate_github

Drama Alert Nation

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	html/changelogs/.all_changelog.yml

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

# Conflicts:
#	code/modules/mob/living/silicon/robot/robot_modules.dm
#	tgui/packages/tgui/public/tgui.bundle.js

Commit:
fixed bad merge conflict

MMMiracles committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summer_ball_2020

# Conflicts:
#	icons/turf/floors.dmi

Commit:
tried to make map airtight, changed around some can_fire configs for the map, added meme

MMMiracles committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summer_ball_2020

Commit:
atmos turns off post setup, similarly to how we normally do it.

Commit:
changed aroudn max integrity, made a working copy of the gondola code. This copy doesn't work and we're using it for testing.

Commit:
extra

Commit:
fixes main branch so it compiles? Not sure why??????

GreyTide2108 committed:
PropHunt Map Update (#58)

* Update Prop Hunt Readme.md

* Map updates

Adds clutter

MMMiracles committed:
i commit thy dock

MMMiracles committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summer_ball_2020

MMMiracles committed:
fix map change overwrites

MMMiracles committed:
Merge pull request #59 from CitrusGender/summer_ball_2020

boardwalk overhaul

Commit:
added gondola boats (kinda functional), set foundation for prophunt get code, and changed the prophunt arena a little bit.

Commit:
changes to air SS initialization.

Commit:
stops runtime and allows for prophunt.dm to spawn in players.

optimumtact committed:
Adds a config setting to deny players with no previous experience (#60)

Adds a configuration options that allows denying users who haven't
gained a requiured amount of living hours on the server previously

This can be used to reduce grief from alt and unused accounts during
major events and or having servers that only have players who have
demonstrated an ability to play over a longer period and not acquire any
bans

optimumtact committed:
Ensure the exp data is loaded before checking time counts (#61)

ishitbyabullet committed:
hey, will this work now

I hope it does.

AnturK committed:
Auto prophunt (#62)

* Basic auto prophunt.

* parametrized hider/searcher numbers

* foolfproofs cross signups a bit

* Hmm yes

* Singup improvements and dead marker for mafia

ishitbyabullet committed:
I DONT THINK THEY'RE READY FOR THIS

JUMP IN THE PIT!!

ishitbyabullet committed:
Merge pull request #63 from ishitbyabullet/pleasework

I'm Untouchable

ishitbyabullet committed:
SPOOKED YA

RATTLE ME BONES

ishitbyabullet committed:
S P O O K E D Y A

BOO!

ishitbyabullet committed:
House of Spooks

Spooked ya!

ishitbyabullet committed:
Muhahahaha

WE'RE REALLY AT RISK NOW, BOYS

Commit:
trees won't change their icon state if there's already a premade one now.

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	html/changelogs/.all_changelog.yml

ishitbyabullet committed:
Please send me Mr. Bones' lua files!

Yeah, I'm just going to have to take down the haunted house for not showing it.

ishitbyabullet committed:
Well, I may as well give my map progress in

No one else made any changes today. Might as well do this.

ishitbyabullet committed:
Merge pull request #64 from ishitbyabullet/pleasework

Wild Ride Code

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

# Conflicts:
#	code/game/objects/structures/tables_racks.dm
#	html/changelogs/.all_changelog.yml
#	tgui/packages/tgui/public/tgui.bundle.js

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

Commit:
deletes your teleportation scroll if you enter the bones room.

welcome to the wild ride.

Commit:
gondolas now say random nonsense when turning

Commit:
slight mistake on the last one.

Commit:
okay this time it works I promise

Commit:
removed prophunt bloat code, changed scrolls to take the first spot of people's inventory as security/assistant.

EOBGames committed:
lots of good shit

ishitbyabullet committed:
Merge pull request #65 from EOBGames/summerball7

Lots of Good Shit

AnturK committed:
Fixes replacing sand (#66)

AnturK committed:
Signup fixes and improvements (#67)

* New icon, fixes

* some more feedback and stuff

ishitbyabullet committed:
go go go go go

you can find me in the club

ishitbyabullet committed:
Merge pull request #68 from ishitbyabullet/pleasework

Minor Changes

MMMiracles committed:
mr bones voice box

MMMiracles committed:
god i love get_turf(user)

MMMiracles committed:
oppa boba ski

MMMiracles committed:
i fuckin' fixed the conflict once already but I GUESS ILL DO IT AGAIN

MMMiracles committed:
Merge pull request #69 from MMMiracles/mr_bones

mr bones voice box

Commit:
don't need this line since this if check is never used.

GreyTide2108 committed:
Prophunt Small Fix (#70)

* Update Prop Hunt Readme.md

* Map updates

Adds clutter

* Prophunt Exit Variable Fix

Changes a variable in the exit for prophunt to properly call. Also changes the spawn area for the hiders and seekers.

Commit:
changes up map so that mafia minigame can't be teleported into, roulette_machine.dm is now anchored. Refractored gondolas to not use sleep, added some background ambient noise.

ishitbyabullet committed:
I am fucking tired right now.

Here you go, partner.

ishitbyabullet committed:
I am fucking tired right now. (#72)

Here you go, partner.

AnturK committed:
Makes prophunt reset more foolproof (#74)

Fikou committed:
reathshkhgdkhjh (#73)

* reathshkhgdkhjh

* dasadsg

ishitbyabullet committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into pleasework

GreyTide2108 committed:
Prophunt adjustments (#75)

Prevents fireaxes from being taken outside the arena (Fireaxes from closets generate in nullspace, this would be a good fix for the main branch.)

Changes team comp to 5 hiders 1 seeker.

changes hide time to 1 minuet

ishitbyabullet committed:
Jvk1166z.esp

Watch the skies.

GreyTide2108 committed:
Descriptive Prophunt Map Names (#76)

* Prophunt adjustments

Prevents fireaxes from being taken outside the arena (Fireaxes from closets generate in nullspace, this would be a good fix for the main branch.)

Changes team comp to 5 hiders 1 seeker.

changes hide time to 1 minuet

* Map Names

Moves and names Prophunt Maps

ishitbyabullet committed:
Lots of Minor Changes (#77)

* I am fucking tired right now.

Here you go, partner.

* Jvk1166z.esp

Watch the skies.

ishitbyabullet committed:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into pleasework

ishitbyabullet commented:
**WE'VE BEEN SPOTTED**

CitrusGender commented:
conflict should be easy to fix

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	html/changelogs/.all_changelog.yml

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

# Conflicts:
#	html/changelogs/.all_changelog.yml

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

Commit:
adds sprites back that might have gotten removed.

CitrusGender commented:
![image](https://user-images.githubusercontent.com/31262308/84958245-fb0c6f80-b0ca-11ea-8ea2-7154bca4513a.png)

ishitbyabullet committed:
a b c d e f g h i j k

l m n o p q r s t u v w

ATH1909 commented on code/datums/diseases/_disease.dm:
```diff
@@ -34,6 +34,10 @@
 
+#ifdef EVENTMODE
+/datum/disease/New()
+	QDEL_IN(src, 30)
```
damn

ATH1909 commented on code/game/area/areas/away_content.dm:
```diff
@@ -16,7 +16,7 @@ Unused icons for new areas are "awaycontent1" ~ "awaycontent30"
 	requires_power = FALSE
 	has_gravity = STANDARD_GRAVITY
-	ambientsounds = list('sound/ambience/shore.ogg', 'sound/ambience/seag1.ogg','sound/ambience/seag2.ogg','sound/ambience/seag2.ogg','sound/ambience/ambiodd.ogg','sound/ambience/ambinice.ogg')
+	ambientsounds = list('sound/ambience/sea_shanty2.ogg')
```
based

ATH1909 commented on code/game/objects/effects/catroomba.dm:
```diff
@@ -0,0 +1,16 @@
+/obj/effect/catroomba
+	icon = 'icons/effects/catroomba.dmi'
+	name = "cat roomba video"
+	desc = "holy shit"
```
holy shit

ATH1909 commented on code/game/objects/items/scrolls.dm:
```diff
@@ -71,3 +71,34 @@
+	var/list/acceptable_locations = GLOB.teleportlocs
+	for(var/L in acceptable_locations)
+		var/area/T = acceptable_locations[L]
+		if(istype(T, /area/security) || istype(T, /area/shuttle/arrival) || istype(T, /area/crew_quarters/dorms) || istype(T, /area/crew_quarters/theatre/abandoned) || istype(T, /area/bridge/meeting_room/council) || istype(T, /area/crew_quarters/theatre) || istype(T, /area/crew_quarters/kitchen) || istype(T, /area/security/nuke_storage) || istype(T, /area/shuttle/abandoned) || istype(T, /area/crew_quar ...
```
isn't nuke_storage already a subtype of the security area?

ATH1909 commented on code/game/objects/items/scrolls.dm:
```diff
@@ -71,3 +71,34 @@
 		to_chat(user, "The spell matrix was disrupted by something near the destination.")
+
+/obj/item/teleportation_scroll/no_smoke
+	uses = 6
```
maybe bump this up to 999 or something? idk

ATH1909 commented on code/game/objects/structures/beds_chairs/chair.dm:
```diff
@@ -7,7 +7,7 @@
 	buckle_lying = 0 //you sit in a chair, not lay
 	resistance_flags = NONE
-	max_integrity = 250
+	max_integrity = 999
```
hugbox content smh my head my head

ATH1909 commented on code/game/objects/structures/fluff.dm:
```diff
@@ -261,3 +267,12 @@
+
+/obj/structure/fluff/tombofhorrors
+	name = "great green devil"
+	desc = "On the wall before you is a relief statue of a devil formed of mosaic plastics. The face has a huge O of a mouth, inside of which the space is dead black darkness."
```
add some way to interact with this, even if it's just getting your hand annihilated because you stuck it in the devil's mouth like a dumbass

ishitbyabullet replied:
Considered.

actions-user committed:
Automatic changelog compile [ci skip]

ATH1909 commented on code/modules/clothing/glasses/_glasses.dm:
```diff
@@ -334,6 +334,17 @@
+	desc = "What are you wearing this for?"
+	icon_state = "mrbonesglasses"
+	inhand_icon_state = "bigsunglasses"
+	clothing_flags = BLOCKS_SHOVE_KNOCKDOWN
```
I was debating telling you this, but I figured that I'm probably not gonna pick a (fair) fight with Mr. Bones anyway: BLOCKS_SHOVE_KNOCKDOWN does NOT protect you from the gun disarm that occurs when you get shoved twice in a row (even if you haven't been shoved into a wall), so a robust enough gamer/swarm of gamers could provoke Mr. Bones into pulling out his """.357""", then disarm him of it, then shoot him to death with it. Maybe the admins could roll with that, though, and make the new owner of the .357 the new Mr. Bones ("there must always be... a Mr. Bones") or something.

ishitbyabullet commented on code/modules/awaymissions/mission_code/Academy.dm:
```diff
@@ -399,3 +399,37 @@
 	if(is_ghost || !(user.mind in SSticker.mode.wizards))
 		..()
+
+/obj/structure/ladder/unbreakable/rope // i regret to inform you but fikou is, in fact, cringe
```
Not considered.

ishitbyabullet commented on code/modules/mob/living/simple_animal/hostile/skeleton.dm:
```diff
@@ -86,6 +85,27 @@
+	color = rgb(0,255,0)
+	speed = 2
+	speak_chance = 1
+	speak = list("You’ve left and left and found my tomb, and now your soul will die.")
```
That's the original poem.

ishitbyabullet commented on code/modules/ruins/objects_and_mobs/necropolis_gate.dm:
```diff
@@ -1,6 +1,6 @@
-//The necropolis gate is used to call forth Legion from the Necropolis.
+//The necropolis gate is used to piss me off.
```
You would understand if you were there.

ishitbyabullet commented on code/modules/vending/clothesmate.dm:
```diff
@@ -140,3 +146,366 @@
+	contraband = list()
+	premium = list()
+
+/obj/machinery/vending/clothing/secretlizard
```
Nah.

ATH1909 replied:
then why does the vending machine say that it's taking sacrifices of money?

EOBGames replied:
Secret Lizard will collect his dues. He always collects his dues.

ATH1909 commented on code/modules/awaymissions/mission_code/Academy.dm:
```diff
@@ -399,3 +399,37 @@
 	if(is_ghost || !(user.mind in SSticker.mode.wizards))
 		..()
+
+/obj/structure/ladder/unbreakable/rope // i regret to inform you but fikou is, in fact, cringe
```
```suggestion
/obj/structure/ladder/unbreakable/rope // i regret to inform you but fikou is, in fact, kek

```
how could fikou be cringe

he added ca[redacted]mba

ATH1909 commented on _maps/map_files/BallMap/summer_ball.dmm:
```diff
@@ -0,0 +1,3184 @@
+"zK" = (/obj/structure/closet/crate/freezer,/obj/item/storage/cans/sixbeer,/turf/open/floor/wood,/area/awaymission/beach)
+"zL" = (/obj/structure/table/reinforced,/obj/effect/turf_decal/trimline/blue/filled/line{dir = 1},/turf/open/floor/plasteel/white,/area/medical/medbay/lobby)
+"zM" = (/obj/mafia_game_signup,/turf/open/floor/plasteel/grimy,/area/crew_quarters/abandoned_gambling_den)
+"zN" = (/obj/structure/chair/comfy/brown{color = "#66b266"; desc = "Not to be confused with messaging other people."; dir = 2; max_integrity = 9999; name = "DM Chair"},/obj/item/clothing/head/wizard{desc = "DnD swag, 20 d20s in my dicebag."; name = "Hat of Dungeon Mastery +3"; pixel_y = 12},/turf/open/floor/carpet/green,/area/crew_quarters/abandoned_gambling_den)
```
you might wanna use a fake wizard hat here; real ones have armor

EOBGames replied:
given that you can buy armored clothing from the jobdrobes, I don't think it's really that dangerous to include a wizard hat

ATH1909 commented on _maps/map_files/BallMap/summer_ball.dmm:
```diff
@@ -0,0 +1,3184 @@
+"mr" = (/obj/machinery/door/window/brigdoor{dir = 1; name = "Secure Window"},/turf/open/floor/carpet,/area/awaymission/beach)
+"ms" = (/turf/closed/indestructible/riveted,/area/awaymission/beach)
+"mt" = (/obj/effect/turf_decal/trimline/blue/filled/line{dir = 1},/turf/open/floor/plasteel/white,/area/medical/medbay/central)
+"mu" = (/obj/effect/decal/cleanable/dirt,/obj/item/reagent_containers/glass/bottle/adminordrazine{name = "Vial of Adminorazine"},/obj/item/reagent_containers/glass/bottle/adminordrazine{name = "Vial of Adminorazine"},/obj/item/reagent_containers/glass/bottle/adminordrazine{name = "Vial of Adminorazine"},/turf/open/floor/plasteel/rockvault,/area/awaymission/cabin/snowforest/sovietsurface)
```
```suggestion
"mu" = (/obj/effect/decal/cleanable/dirt,/obj/item/reagent_containers/glass/bottle/adminordrazine{name = "Vial of Adminordrazine"},/obj/item/reagent_containers/glass/bottle/adminordrazine{name = "Vial of Adminordrazine"},/obj/item/reagent_containers/glass/bottle/adminordrazine{name = "Vial of Adminordrazine"},/turf/open/floor/plasteel/rockvault,/area/awaymission/cabin/snowforest/sovietsurface)
```

ATH1909 commented on _maps/map_files/BallMap/summer_ball.dmm:
```diff
@@ -0,0 +1,3184 @@
+"lJ" = (/obj/effect/turf_decal/trimline/blue/filled/warning{dir = 1},/obj/structure/chair{dir = 1},/turf/open/floor/plasteel/white,/area/medical/medbay/lobby)
+"lK" = (/obj/structure/railing/corner{icon_state = "railing_corner"; dir = 4},/turf/open/floor/plating/beach/sand,/area/awaymission/beach)
+"lL" = (/obj/structure/spider/stickyweb,/obj/effect/decal/remains/human,/turf/open/floor/plating/ashplanet/rocky,/area/awaymission/cabin/snowforest/sovietsurface)
+"lM" = (/obj/structure/spider/stickyweb,/mob/living/simple_animal/hostile/retaliate/bat{desc = "A fierce companion for any person of power, this spider has been carefully trained by Nanotrasen specialists. Its beady, staring eyes send shivers down your spine."; emote_hear = list("chitters"); faction = list("spiders"); harm_intent_damage = 3; health = 200; icon_dead = "guard_dead"; icon_gib = "gua ...
```
why the fuck are you using a bat as the base creature here when what you clearly want is Sergeant Araneus?

ishitbyabullet replied:
because sergeant araneus is actually a bat for some reason.

tralezab replied:
that got fixed jeril

EOBGames replied:
it's in as a reference then

ATH1909 replied:
> because sergeant araneus is actually a bat for some reason.

what the damn hell

ishitbyabullet committed:
story of ed and eddy

im not really sure that you're ready

ishitbyabullet committed:
Merge pull request #78 from ishitbyabullet/pleasework

Adds sporks.

MMMiracles committed:
spooky skeletons!!! (#79)

doot

tralezab committed:
AHA I CONTRIBUTED CAN'T SAY I DIDN'T

tralezab committed:
Revert "AHA I CONTRIBUTED CAN'T SAY I DIDN'T"

This reverts commit 6ea5d01078f77a30e3f59a8ee98f2a2142c9ef93.

tralezab committed:
real fix sorry i got excited

Fikou commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -194,6 +194,20 @@
+	flags_inv = HIDEEARS|HIDEHAIR
+	icon_state = "mrbones"
+	inhand_icon_state = "thunderdome"
+	armor = list("melee" = 80, "bullet" = 80, "laser" = 50, "energy" = 50, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 90)
```
ok but how do you want to get bombs

ATH1909 replied:
oh, wait, is just his hat armored? if so, then keeping it like this is fine

AnturK commented:
Doesn't need to be open for anything

trollbreeder committed:
Merge pull request #80 from tralezab/patch-92

Fixes mafia (thanks anturk)

Fikou committed:
adds eye statue sprite

trollbreeder committed:
Merge pull request #82 from Fikou/eyestatue

adds eye statue sprite

ishitbyabullet committed:
WELCOME TO THE MAIN EVENT

WATCH THIS - THE CROWD LOVES IT

ishitbyabullet committed:
Merge pull request #85 from ishitbyabullet/pleasework

Removes unvectored tables.

AnturK committed:
Hud signup button (#84)

* hud signup buttons - no area changes.

* yes

tralezab committed:
Prophunt projectors, win and lose conditions, poki mane poggers sex, and more! (#81)

* Update prophunt.dm

* okay... uh... we should test this...

* suggestions

AnturK committed:
Yes (#87)

ishitbyabullet committed:
End Credits

When you're taught through feelings
When you're taught through feelings
When you're taught through feelings
When you're taught through feelings
Destiny flying high above
All I know is that you can Realise (Realise)
Destiny who cares (Destiny who cares)
As it turns around (as it turns around)
And I know that it descends down on me
It's just another day
The shame is gone
Hard to believe
That I've let it go
Let it go
Let it go
Destiny can't replace my life
Scary shadows of my past are alive (are alive)
Destiny who cares (Destiny who cares)
As it turns around (as it turns around)
And I know that it descends with a smile
It's just another day
The shame is gone
It's Hard to believe
That I let it go away
It's just a melody
It bleeds in me
Hard to believe
That I let it go
When you're taught through feelings
When you're taught through feelings
When you're taught through feelings
When you're taught through feelings
It's just another day
The shame is gone
It's Hard to believe
That I let it go away
It's just a melody
It bleeds in me
Hard to believe
That I let it go
Let it go
Let it go

ishitbyabullet committed:
Merge pull request #88 from ishitbyabullet/pleasework

Fikou and Lizards: 5th edition

CitrusGender commented:
but it doesn't show up in the TESTMERGE LIST WHEN ITS CLOSED REEEEE

tralezab committed:
jesus (#89)

trollbreeder committed:
Paramedic, where are you storing that staff? (#90)

I keep it up my arse.

Co-authored-by: trollbreeder <kkk>

tralezab committed:
mafia judgement phases, better voting, no day 1 lynch

tralezab committed:
Merge remote-tracking branch 'upstream/summerball2020' into mafia_polish

actions-user committed:
Automatic changelog compile [ci skip]

tralezab committed:
more shit

tralezab committed:
alright that's all i got

ishitbyabullet committed:
HURRY UP THE PR

FOR FUCKSAKE HURRY UP

ishitbyabullet committed:
Merge pull request #92 from ishitbyabullet/pleasework

Unregistered polygons have been fixed.

GreyTide2108 committed:
Prophunt Map Changes

Removed: Power Cables, Pipes, Door Access issues

EOBGames committed:
important changes

temple is fixed boss

ishitbyabullet committed:
Merge pull request #91 from tralezab/mafia_polish

mafia judgement phase, better phase timing, no day 1 lynch, and more

ishitbyabullet committed:
Merge pull request #93 from GreyTide2108/summerball2020

Prophunt Map Changes

EOBGames committed:
jungletemple better

ishitbyabullet committed:
Merge pull request #95 from EOBGames/summerball9

The Temple Reborn

Commit:
you can no longer suicide.

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

ATH1909 commented:
I noticed that a certain thing said "Follow the Blue Clues." instead of "Follow the Blue's Clues."; is that intentional?

ishitbyabullet committed:
took 2 hours to make this change

the wonders of a funny conversation!

ishitbyabullet committed:
Merge pull request #98 from ishitbyabullet/pleasework

Disables Unintended Length Shifting

tralezab committed:
Solo Antag Refactor + More Solo Antags (#96)

* Solo Antag Refactor + More Solo Antags

* testing

* obsessed does it on night one, fugitive gets his damn color

* cleanup

* frick

* oh my god thank lord i caugth this

tralezab committed:
le final mafia pr

Fikou committed:
Create f.dm

Fikou committed:
adds the mapper lair

Fikou committed:
Delete f.dm

trollbreeder committed:
Merge pull request #100 from Fikou/patch-26

adds the mapper lair

ishitbyabullet committed:
Merge pull request #99 from tralezab/mafia-teleporting

Mafia now teleports you to town square when you are put on trial

ishitbyabullet committed:
through the gates of hell

im tired of making joke descriptions today

ishitbyabullet committed:
Merge pull request #101 from ishitbyabullet/pleasework

Removes Unfiltered Charon Rifts

Fikou committed:
no no no no NO (#102)

Commit:
qdels chameleon projector in prophunt only if caught. Shouldn't get caught in a qdel loop now.

Commit:
extra fix so that people get booted out of the arena.

Fikou committed:
you can no longer dig up grass (#103)

* you can no longer dig up grass

* map fixes

* map fixes

CitrusGender commented:
the PR isn't updating when it's not open.

trollbreeder commented on code/modules/mob/living/silicon/robot/robot_modules.dm:
```diff
@@ -300,31 +291,17 @@
 	basic_modules = list(
-		/obj/item/assembly/flash/cyborg,
-		/obj/item/borg/sight/meson,
-		/obj/item/construction/rcd/borg,
```
yes we need to defang borgs else we get screwdriver griefers

Fikou commented:
please dont murder me in my sleep it just needs commits updated...

ishitbyabullet commented:
Stop opening and closing it.

Commit:
removes clothing type without icon from vending machine.

ishitbyabullet committed:
TLOU2

absolute trash dont buy it

ishitbyabullet committed:
Seriously, don't.

I'm warning you homies.

ishitbyabullet committed:
Merge pull request #105 from ishitbyabullet/pleasework

Removes Selection Island

ishitbyabullet committed:
Revert "temporary change"

This reverts commit 3f915c8c319ba4e749eb8343ca271cd505312052.

ishitbyabullet committed:
temporary change

will be reverted later

tralezab committed:
done? we hope

ishitbyabullet committed:
Merge pull request #107 from tralezab/more-mafia-fixes,-more-mafia-improvements

Even more mafia improvements

AnturK committed:
Makes mafia and prophunt start on auto (#104)

* Makes mafia and prophunt start on auto

* huh that was broken before

* remove test dmm

tralezab committed:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

ishitbyabullet committed:
Merge pull request #108 from tralezab/gogogogo

Small mafia oversight

GreyTide2108 committed:
Prophunt 1840 EST Push / T - 20:20:00 (#106)

* Prophunt Last Minuet Homework

It's not late if it's 2 minuets before it's due.

Lots of bug fixes to prophunt maps and prophunt lobby.

* Cash update

* Take it

Commit:
Merge remote-tracking branch 'upstream/master'

# Conflicts:
#	html/changelogs/.all_changelog.yml

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

# Conflicts:
#	tgui/packages/tgui/public/tgui.bundle.js

CitrusGender commented:
I'll open/close so it's not on the tracker.

ishitbyabullet commented:
**AAAAAAAAAAAAAAA**

CitrusGender commented:
yeah I don't want to constantly open and close it so if I can just leave it open that'd be really appreciated.

Commit:
removes stuff that got removed.

optimumtact committed:
Add admin bypass to whitelist (#109)

actions-user committed:
Automatic changelog compile [ci skip]

optimumtact committed:
Actually check for failure case (#110)

Success case returns null for some reason

ishitbyabullet committed:
The Ballad of Buster Scruggs

You can't be topdog forever.

ishitbyabullet committed:
Merge pull request #111 from ishitbyabullet/pleasework

Deletes Unregistered Chromobones

MMMiracles committed:
Lamplamplamp (#112)

* hold this please

* thanks for holding that

EOBGames committed:
off to you, ayyman

Commit:
whitelist should work now, removed some clothing that was basetypes, changed casino ambient noise, added sprites from domi to coffee bot that follows you around and throws coffee at you.

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

ishitbyabullet committed:
Merge pull request #114 from EOBGames/summerball10

it's 6am please send help

Commit:
removes outdated types from prophunt map and map that doesn't work.

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

ishitbyabullet committed:
i dont really know what else to add

i guess we're done

CitrusGender commented:
CRUNCH TIME CRUNCH TIME CRUNCH TIME

ishitbyabullet committed:
Merge pull request #115 from ishitbyabullet/pleasework

Removes Communication Consoles

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

Commit:
mapmerges and adds coffeebot route.

tralezab committed:
1, 2, 3, 4, 5 (#116)

bobbah committed:
Adds Interview System Add-on For Allowlist System (#113)

* oranges stuff

* some cleanup

* stage one

* wip

* wip

* yoo lemme fix that real quick

* cleanup

* move interview into module

* a stately home for our interviews

* base implementation of stat panel

* add approval/denial

* cooldown and other stuff

* janitorial functions

* now we have working refresh

* documentation

* tgui bundle rebuild with stuff from ball

* lets keep those answers short ladies!

* oops

* add interview manager ui

CitrusGender committed:
Blackjack (#97)

* start of my meme...

* game should be good, adds noises and icons

* bringing it up to date

* whatever

* brings blackjack up to date, probably needs more testing

* adds blackjack bots to ball, adds a few fixes and stops cards from showing their values unless they're involved in blackjack.

Co-authored-by: Citrus <a_anakin96@hotmail.com>

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

CitrusGender committed:
Merge branch 'master' into summerball2020

CitrusGender commented:
oh god

CitrusGender commented:
![image](https://user-images.githubusercontent.com/31262308/85196105-d4eef700-b2a5-11ea-8777-08c675b3570a.png)

optimumtact commented:
wow I could merge it

Commit:
you cannot teleport out of the bone zone.

Commit:
Merge remote-tracking branch 'origin/master' into summerball2020

Commit:
sets exp correctly.

bobbah committed:
fix (#120)

Commit:
stops the lag by stopping maptext from showing.

Commit:
Merge branch 'summerball2020' of https://github.com/CitrusGender/tgstation into summerball2020

Cyberboss commented:
We're done with this, right?
